crates/rooch-rpc-client/src/lib.rs

```diff
@@ -59,7 +62,8 @@ impl ClientBuilder {
             .build(http)?;
 
         Ok(Client {
-            rpc: Arc::new(RpcClient { http: http_client }),
+            rpc: Arc::new(RpcClient { http: http_client.clone() }),
+            ethereum_rpc: Arc::new(EthereumRpcClient { http: http_client.clone() }),
```

Comment:
Same as above, we do not need the `EthereumRpcClient`.

Reply:
Ok.